packages/storybook-vue/stories/3_components/table/Table.stories.mdx

```diff
@@ -247,6 +248,7 @@ scale-table {
   --box-shadow-th-sortable-focus: inset 0 0 0 var(--scl-spacing-2) var(
       --scl-color-focus
     );
+  --background-secondary-tbody: var(--scl-color-grey-0);
```

Comment:
I would probably name this `--background-tr-striped` :)

Reply:
Adapted in last commit :)